src/components/PassphraseDialog/PassphraseDialog.scss

```diff
@@ -36,17 +38,18 @@ $color-backlog-blue--dark: #5c8fff;
 
 .form__icon {
   display: none;
-  color: $color-backlog-blue--light;
-  opacity: 50%;
-  height: $icon--large;
-  width: $icon--large;
+  color: $color-backlog-blue--dark;
```

Comment:
Why have you changed the color from backlog blue with opacity: 50% to backlog blue dark?

Reply:
because that is the color in the design file. And having it a fixed color at 100% opacity makes more sense imo, because this way it's not depending on the background color